jinja2/react_base.html

```diff
@@ -46,6 +46,10 @@
     {% include "includes/google_analytics.html" %}
   {% endblock %}
 
+  {% block speedcurve_lux %}
+    {% include "includes/speedcurve-lux.html" %}
+  {% endblock %}
```

Comment:
I find jinja templates confusing, so I removed as much of the inheritance as I could from kuma/wiki/jinja2/wiki/react_document.html.  Which means that it no longer inherits from react_base.html, and you'll need to add these lines explicitly to react_document.html.

react_base.html is still used by the react homepage in kuma/landing/jinja2/landing, but I might be changing that soon....